onnxruntime/test/optimizer/graph_transform_utils_test.cc

```diff
@@ -42,7 +42,8 @@ TEST(GraphTransformerUtilsTests, TestGenerateGraphTransformers) {
   std::vector<std::string> custom_list = {l1_rule1, l1_transformer, l2_transformer};
 
   auto transformers = optimizer_utils::GenerateTransformers(TransformerLevel::Level1, {}, custom_list);
-  ASSERT_TRUE(transformers.size() == 2);
+  //ASSERT_TRUE(transformers.size() == 2);
+  ASSERT_TRUE(transformers.size() == 1);  // TODO hack - temporarily disabled constant folding
```

Comment:
Sherlock: This is a hack. need to fix this
Scott: Just checking that the plan is to fix this prior to checking this PR in.
Sherlock: Yes. Will fix before check in - https://msdata.visualstudio.com/Vienna/_boards/board/t/Training%20Runtime/Features/?fullScreen=true&workitem=604684
to track this

Reply:
reverted